test/model/content_test.dart

```diff
@@ -301,6 +301,17 @@ class ContentExample {
         '\n</code></pre></div>'),
     ]);
 
+  static const codeBlockFollowedByMultipleLineBreaks = ContentExample(
+    'blank text nodes after code blocks',
+    '    code block.\n\nsome content',
+    // https://chat.zulip.org/#narrow/stream/7-test-here/topic/2.20new.20lines/near/1774823
```

Comment:
nit: leave out topic to reduce visual clutter, as we did in #603 and as specified in #617, so:

```dart
    // https://chat.zulip.org/#narrow/stream/7-test-here/near/1774823
```